src/components/search/Search.tsx

```diff
@@ -0,0 +1,88 @@
+import {Text, TextInput, View, FlatList} from 'react-native';
+import {useEffect, useState} from 'react';
+import Icon from 'react-native-vector-icons/Feather';
+
+import {theme} from '@theme/themes';
```

Comment:
fix your import for `theme`, there are 3 mentions of "theme" this can be refactored